tokio/src/runtime/context.rs

```diff
@@ -198,3 +183,24 @@ cfg_rt! {
         }
     }
 }
+
+cfg_rt_or_time! {
```

Comment:
Is this the correct macro ? The code below cares only about `rt` but not for `time`.

Reply:
This method is also called by the timer.